python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -308,3 +382,48 @@ def _inplace_subtract(node: ResourceDict, resources: ResourceDict) -> None:
         assert k in node, (k, node)
         node[k] -= v
         assert node[k] >= 0.0, (node, k, v)
+
+
+def _inplace_add(a: Dict, b: Dict) -> None:
+    for k, v in b.items():
+        a[k] += v
+
+
+def placement_groups_to_resource_demands(placement_group_load: List[
+        PlacementGroupTableData]):
+    """Preprocess placement group requests into regular resource demand vectors
+    when possible.
```

Comment:
I think the template is that this should be a single line.

Reply:
That would break lint